sdk/keyvault/azure-security-keyvault/README.md

```diff
@@ -1,13 +1,116 @@
+```
+## Key concepts
+### Vault
+  A user can create a key vault and gain full access and control over it. When using Key Vault, application developers no longer need to store security information in their application. Not having to store security information in applications eliminates the need to make this information part of the code.
+
+### Vault Client:
+A vault client performs interactions with the Keys and Secrets client for creating and getting an instance of the Keys/Secrets client. An asynchronous and synchronous, VaultClient, client exists in the SDK allowing for selection of a client based on an application's use case.
+
+## Examples
+The following sections provide several code snippets covering some of the most common Key Vault related tasks, including:
+* [Create a Secret Client](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/keyvault/azure-security-keyvault#create-a-secret-client)
+* [Create a Key Client](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/keyvault/azure-security-keyvault#create-a-key-client)
+
+### Create a Secret Client
+```python
+    from azure.common.credentials import ServicePrincipalCredentials
+    from azure.security.keyvault import VaultClient
+
+    credentials = ServicePrincipalCredentials(
+        client_id=client_id, secret=client_secret, tenant=tenant_id, resource="https://vault.azure.net"
+    )
+
+    # Create a new Vault client using Azure credentials
+    vault_client = VaultClient(vault_url=vault_url, credentials=credentials)
+    # retrieves an instance of Secret Client
+    secret_client = vault_client.secrets
+```
+
+### Create a Key Client
+```python
+    from azure.common.credentials import ServicePrincipalCredentials
+    from azure.security.keyvault import VaultClient
+
+    credentials = ServicePrincipalCredentials(
+        client_id=client_id, secret=client_secret, tenant=tenant_id, resource="https://vault.azure.net"
+    )
+
+    # Create a new Vault client using Azure credentials
+    vault_client = VaultClient(vault_url=vault_url, credentials=credentials)
+    # retrieves an instance of Key Client
+    key_client = vault_client.keys
+```
+
+## Troubleshooting
+### General
+The Key Vault APIs generate exceptions that can fall into one of the azure-core defined [exceptions](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/core/azure-core/azure/core/exceptions.py).
+
+### Logging
+This SDK uses Python standard logging library. You can configure logging print out debugging information to the stdout or anywhere you want.
+
+```python 
+import logging
+logging.basicConfig(level=logging.DEBUG)
+```
+Http request and response details are printed to stdout with this logging config.
+
+## Next steps
+* Set and retrieve a secret [TODO: add link]
+* Set and retrieve a key [TODO: add link]
+
+###  Documentation
+Reference documentation is available at docs.microsoft.com/python/api/azure-security-keyvault
```

Comment:
it is expected to have the new documentation at this location, but it does not exist currently?